lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -201,10 +199,70 @@ void Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
         calculateConvPoolOutputDims(idim.h, idim.w, kernels, strides, pads);
     std::array<size_t, 4> outDims = {
         {idim.n, outSz.first, outSz.second, depth}};
-    auto outTy = G_.getParent()->uniqueType(ElemKind::FloatTy, outDims);
+
+    TypeRef outTy;
+    Constant *filter;
+    Constant *bias;
+    if (typeName == "Conv") {
+      // Construct the Bias field.
+      Tensor biasTensor(ElemKind::FloatTy, {depth});
+      biasTensor.zero();
+
+      // Check if we have a serialized bias vector.
+      if (op.input_size() > 2) {
+        auto &biasTensorName = op.input(2);
+        if (tensors_.count(biasTensorName)) {
+          // Load the serialized bias vector.
+          Tensor *b = getTensorByName(biasTensorName);
+          biasTensor.assign(b);
+        }
+      }
+      outTy = G_.getParent()->uniqueType(ElemKind::FloatTy, outDims);
+      filter = G_.getParent()->createConstant("conv.filter", wtag);
+      bias = G_.getParent()->createConstant("conv.bias", biasTensor);
+    } else {
+      assert(dict.count("Y_zero_point") &&
+             "missing zero point for quantzied type");
+      assert(dict.count("Y_scale") && "missing Y_scale for quantzied type");
+      // Construct the Bias field.
+      Tensor biasTensor(ElemKind::Int32QTy, {depth}, 1.0, 0);
+      biasTensor.zero();
+      // Check if we have a serialized bias vector.
+      if (op.input_size() > 2) {
+        auto &biasTensorName = op.input(2);
+        if (tensors_.count(biasTensorName)) {
+          // Load the serialized bias vector.
+          Tensor *b = getTensorByName(biasTensorName);
+          biasTensor.assign(b);
+        }
+      }
+      float scale = loadFloat(dict["Y_scale"]);
+      int32_t offset = loadInt(dict["Y_zero_point"]);
+      if (typeName == "Int8ConvRelu")
+        offset += OFFSETSHIFT;
+      outTy = G_.getParent()->uniqueType(ElemKind::Int8QTy, outDims, scale,
+                                         offset - OFFSETSHIFT);
+
+      // Construct the quantized Filter and bias field.
+      filter = G_.getParent()->createConstant(
+          ElemKind::Int8QTy, wtag.dims(), wtag.getType().getScale(),
+          wtag.getType().getOffset(), "conv.filter");
+      filter->assign(&wtag);
+      bias = G_.getParent()->createConstant(
+          ElemKind::Int32QTy, biasTensor.dims(),
+          biasTensor.getType().getScale(), biasTensor.getType().getOffset(),
+          "conv.bias");
+      bias->assign(&biasTensor);
+    }
 
     Node *node = G_.createConv(opName, tr, filter, bias, outTy, kernels,
                                strides, pads, group);
+    if (typeName == "Int8ConvRelu") {
```

Comment:
similarly how you handle `Int8SumRelu` we should be able to handle Int8ConvRelu without additional Relu node.